test/elixir/run.cmd

```diff
@@ -1,5 +1,7 @@
 @ECHO OFF
 
 cd %~dp0
+call mix local.hex --force
+call mix local.rebar --force
```

Comment:
This change, and the one to `test/elixir/run`, will be necessary to make the Elixir tests work in a CI environment - otherwise an interactive prompt is raised to first install hex, then to install rebar.